### ISSUE TITLE
Name Idea

### ISSUE THREAD

**Issue by nteetor** [travisbyrum/clinkr#1]
This idea came to me this morning based on your description of the package in the README, "... Command line interfaces in R".

Command line interface is often abbreviated as _cli_, so I thought why not thought "clir". The pronunciation would be _clear_. After more consideration I felt "clir" would not seem obviously pronounced as _clear_. 

However, "cliir", while a little zany, looks more like _clear_ and would stand for "Command Line Interfaces In R". Food for thought.

p.s. There's also tons of room for puns with a package name that sounds like _clear_

**Comment by nteetor** [comment]
@travisbyrum Do you get notifications for issues?

**Comment by travisbyrum** [comment]
Yeah I get notifications but it's fine.  I actually like clir so we can switch to that.  I'm also thinking of using your hashmap class as a way to store all of the information for each argument.  I think that might be kinda cool.

**Comment by nteetor** [comment]
You have the code for the HashMap class?? I was looking through my repos the other day, but couldn't find that code. Was HashMap a part of merrem?

re: name, there's no pressure to use the name, but I wanted to make sure you had seen my suggestion. Since I'm not contributing code yet, I gotta make sure I do _something_.
